recipe-client-addon/content/about-studies/common.js

```diff
@@ -0,0 +1,149 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+"use strict";
+/* eslint-disable no-unused-vars */
```

Comment:
The [no-unused-vars](http://eslint.org/docs/rules/no-unused-vars) docs say you can use an `exported` comment block for this, but for reasons unknown to me this doesn't work with the eslint setup we have. I spent like an hour yesterday trying to get past it and failing, and this comment is a sign of my failure. :P

I can file a followup issue for `exported` comment blocks not working, would that be a good compromise?